src/CoVoX/API/Settings/SecretsHelper.cs

```diff
@@ -1,20 +1,47 @@
 ﻿using System;
+using System.Linq;
 using Microsoft.Extensions.Configuration;
 
 namespace Covox
 {
-    public class SecretsHelper
+    internal class SecretsHelper
     {
-        public static AzureConfiguration GetSecrets()
+        /// <summary>
+        /// Reads AzureConfiguration from secrets.json file located in the application folder.
+        /// </summary>
+        /// <param name="path">Path to the secrets.json file.</param>
+        /// <returns>AzureConfiguration with values read from secrets.json located in path.</returns>
+        internal static AzureConfiguration GetAzureConfigurationFromSecretsJson(string path)
         {
             IConfiguration config = new ConfigurationBuilder()
                 .SetBasePath(AppDomain.CurrentDomain.BaseDirectory)
-                .AddJsonFile("secrets.json")
+                .AddJsonFile(path)
                 .Build();
 
             var section = config.GetSection(nameof(AzureConfiguration));
 
             return section.Get<AzureConfiguration>();
         }
+
+        /// <summary>
+        /// Reads AzureConfiguration from COVOX_AZURE_CONFIG environment variable. Variable format must be subscriptionKey:region.
+        /// </summary>
+        /// <param name="variable">Name of the environment variable where the keys are stored.</param>
+        /// <returns>AzureConfiguration with values read from environment variable.</returns>
+        internal static AzureConfiguration GetAzureConfigurationFromEnvironmentVariables(string variable)
```

Comment:
Given the specificity of this operation, I'd put it directly in the `AzureConfiguration.FromEnvironmentVariable` method.
Or maybe we can make it generic? read from envvar + split via separator.